net/instaweb/rewriter/cache_extender_test.cc

```diff
@@ -1157,6 +1157,44 @@ TEST_F(CacheExtenderTest, VaryOrigin) {
                    StringPrintf(kCssFormat, cache_extended_css.c_str()));
 }
 
+TEST_F(CacheExtenderTest, RenderCsp) {
+  InitTest(100);
```

Comment:
It's not clear to me what '100' means here (and in the lines below the same number shows up, should it maybe be constantized?)